fiware/src/SystemHandle.hpp

```diff
@@ -32,7 +33,7 @@ namespace fiware {
 
 class NGSIV2Connector;
 
-class SystemHandle : public virtual TopicSystem
+class SYSTEM_HANDLE_EXPORT SystemHandle : public virtual TopicSystem
```

Comment:
What happens if any other class or utility needs to be exported. Could be a too restrictive word? What about `SOSS_EXPORT` as a more generic name?

Reply:
SOSS_EXPORT is the word used in soss-core to export symbols. SYSTEM_HANDLE_EXPORT is the word used in all classes in system handlers that you want to export symbols. They are different because when you compile a SystemHandle you have to import core symbols also when you compile core you have to export that symbols. In the other hand, if you are compiling a system handler you only want to export that symbols in order to make a dll valid but nobody will link with that library. It just will be loaded at runtime by soss app. 

The name is refered to the ecosystem when you are coding, not only to system handler class. However, we could change the name to something like SOSS_ONLY_EXPORT.

Reply:
I realized the difference in the soss core PR. Good point, I Like it!